File: WebAPI/WebAPI.Models/Models/Models/Epic.cs
using System;

namespace WebAPI.Models.Models.Models
{
    public class Epic
    {
        public Guid EpicId { get; set; }
        
        public Guid ProjectId { get; set; }
        
        public string EpicName { get; set; }
        
        public string EpicDescription { get; set; }
        
        public DateTime StartDate { get; set; }
        
        public DateTime EndDate { get; set; }
        
        public DateTime CreationDate { get; set; }
        
        public bool IsDeleted { get; set; }
    }
}

File: web/src/mappers/epic.ts
import { IEpic, IEpicSimpleModel } from '../types/epic';

export const mapToEpicModel = (data): IEpic => ({
    epicId: data.epicId,
    epicName: data.epicName,
    epicDescription: data.epicDescription,
    startDate: data.startDate,
    endDate: data.endDate,
    projectId: data.projectId,
    creationDate: data.creationDate,
});

export const mapToEpicSimpleModel = (data): IEpicSimpleModel => ({
    epicId: data.epicId,
    epicName: data.epicName,
    startDate: data.startDate,
    endDate: data.endDate,
});
